web/src/components/customcomponents/loader/CenteringWrapper.tsx

```diff
@@ -0,0 +1,37 @@
+import { Box } from "@chakra-ui/react";
+import classnames from "classnames";
+import * as React from "react";
+
+import "@/components/customcomponents/loader/centering-wrapper.scss";
+
+type Props = {
+  className?: string;
+  vertical?: boolean;
+  children: React.ReactNode;
+};
+
+export class CenteringWrapper extends React.Component<Props> {
+  render() {
```

Comment:
Its copied from singleJS might need to capture how it works

But better to keep it same as our singleJS

Reply:
But do we wan't to do this and override change from singleJS or we wan't to keep it same

As we may need to every time do this in case of changes

Reply:
My guess is the only reason why this is still a class component in single-js is because nobody has touched it in quite some time (last change was from 2 years ago). If someone was to update this, they would probably get the suggestion to change it to a functional component. Either way, it hasn't been changed in a long time, and I doubt we would update the RTDM app just because of it.

So you can update it to a functional component, as this is what we push for now in Helios :slightly_smiling_face:

Reply:
Git it let me make the change